offload/unittests/OffloadAPI/kernel/olLaunchKernel.cpp

```diff
@@ -104,6 +104,29 @@ TEST_P(olLaunchKernelFooTest, Success) {
   ASSERT_SUCCESS(olMemFree(Mem));
 }
 
+TEST_P(olLaunchKernelFooTest, SuccessThreaded) {
```

Comment:
I'd love to be able to add an `OFFLOAD_TEST_THREADED_P` macro so that you'd get threaded and non-threaded tests "for free" without copy-pasting the test body. But I can't think of a good way of actually implementing that with gtest, anyone have any ideas?